### ISSUE TITLE
Fix Toolbar to top button: indicate the state to assistive technologies

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#5700]
Similar to #5699 and reported by one of the accessibility testers:

• When the option “Fix toolbar to top” option is selected, I was expecting to hear “Fix toolbar to top, selected”. Instead, the menu is closed.
• If I open the menu, when the focus is on “Fix toolbar to top” option, the state “selected” is not announced. There is a visual feedback (a check icon) but none for screen readers.

I guess for this button the most appropriate ARIA attribute is aria-pressed https://www.w3.org/TR/wai-aria-1.1/#aria-pressed as aria-selected is not used for buttons https://www.w3.org/TR/wai-aria-1.1/#aria-selected

Also, it would be nice to add a message to confirm that the switch was made.
